pom.xml

```diff
@@ -207,6 +207,17 @@
 			<artifactId>bcprov-jdk15on</artifactId>
 			<version>1.59</version>
 		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito2</artifactId>
+			<version>2.0.0-beta.5</version>

```

Comment:
Do you have to use a beta version for powermock here?

Reply:
Updated to 2.0.2